pkg/adapter/nimbus-kubearmor/processor/kspbuilder.go

```diff
@@ -268,31 +268,25 @@ func disallowChRoot() kubearmorv1.KubeArmorPolicy {
 func disallowBinaries() kubearmorv1.KubeArmorPolicy { // ref: https://www.tenable.com/audits/items/search?q=noexec&sort=&page=1
 	return kubearmorv1.KubeArmorPolicy{
 		Spec: kubearmorv1.KubeArmorPolicySpec{
-			File: kubearmorv1.FileType{
-				MatchDirectories: []kubearmorv1.FileDirectoryType{
+			Process: kubearmorv1.ProcessType{
+				MatchPaths: []kubearmorv1.ProcessPathType{
 					{
-						Directory: "/var/tmp/",
-						Recursive: true,
```

Comment:
The recursive option can be applied for process policies also. Please check the example on 

https://docs.kubearmor.io/kubearmor/documentation/security_policy_examples

Reply:
done @shivaccuknox